source/examples/xgboost-rf-gpu-cpu-benchmark/notebook.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
<p>Format this cell (and other command cells) using the Markdown code block.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>